tests/bluetooth/host/keys/bt_keys_get_addr/src/test_suite_full_list_overwrite_oldest.c

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include "testing_common_defs.h"
+#include <addr.h>
+#include <keys.h>
+#include "mocks/conn.h"
+#include "mocks/hci_core.h"
+#include "mocks/keys_help_utils.h"
+#include "mocks/hci_core_expects.h"
+
+/* This LUT contains different combinations of ID and Address pairs */
+extern const struct id_addr_pair testing_id_addr_pair_lut[CONFIG_BT_MAX_PAIRED];
+
+/* This list holds returned references while filling keys pool */
+extern struct bt_keys *returned_keys_refs[CONFIG_BT_MAX_PAIRED];
+
+/* Pointer to the current set of oldest testing parameter */
+struct id_addr_pair const *oldest_params_vector;
+
+/* This call back will be called when bt_unpair() is called with the same parameters
+ * It will clear the key slot with the assigned to the 'id'
```

Comment:
This sentence doesn't make sense to me. Reading the callback, what it does is get the first `(id, addr)` match and clear it. So it doesn't just depend on `id`, but also `addr`.
I would phrase it `Clears the first key slot that matches its arguments`.

Reply:
I followed your suggestion to remove the comment